pandas/io/formats/format.py

```diff
@@ -1797,17 +1797,13 @@ def get_format_timedelta64(
 
     If box, then show the return in quotes
     """
-    values_int = values.view(np.int64)
+    values_int = cast(np.ndarray, values.view(np.int64))
```

Comment:
cleaner to do `np.asarray(values).view(np.int64)`?

Reply:
changed